src/components/TimeseriesSearch/TimeseriesSearch.tsx

```diff
@@ -34,8 +35,25 @@ const CenteredSpin = styled(Spin)`
   }
 `;
 
+const ButtonRow = styled.div`
+  margin-top: 5px;
+`;
+
+const SelectAllButton = styled((props: NativeButtonProps) => (
```

Comment:
Do we need styled component here if we don't provide any styles to it?

Reply:
Removed